src/main/kotlin/no/nav/syfo/infrastructure/azuread/AzureAdClient.kt

```diff
@@ -51,6 +74,8 @@ class AzureAdClient(
     }
 
     companion object {
+        private const val CACHE_AZUREAD_TOKEN_SYSTEM_KEY_PREFIX = "azuread-token-system-"
+        private val cache = ConcurrentHashMap<String, AzureAdToken>()
```

Comment:
Vi vil nok heller bruke Redis enn å bruke en sånn hashmap som bare lever i én pod.

Reply:
Føles litt overkill med en redis-instans bare for å cache dette. Kanskje vente med det til vi evt ser at det trengs.